packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -112,9 +112,10 @@ export const run = (includeInternalScripts = false) => {
     program
       .command('component:create')
       .description(
-        'Scaffold React components. Optionally add test, story and .mdx files. The components are created in the same dir the script is run from.'
+        '[deprecated] Scaffold React components. Optionally add test, story and .mdx files. The components are created in the same dir the script is run from.'
       )
       .action(async () => {
+        chalk.yellow.bold(`⚠️ This command is deprecated , will be removed in v10 and no support will be provided. ⚠️`);
```

Comment:
```suggestion
        chalk.yellow.bold(`⚠️ This command is deprecated and will be removed in v10. No further support will be provided. ⚠️`);
```

Maybe we should point them to [something else](https://www.npmjs.com/search?q=keywords%3Ascaffold%20react%20component) to use and add something like `if you were reliant on this command we recommend https://www.npmjs.com/package/react-gen-component` 🤷